### PR TITLE
Initial components

### PR DESCRIPTION
**Pull request by mim24sd**
In this pull request I want to make :
1. initial components

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "music-app",
+  "description": "This is a small music react app",
+  "scripts": {
+    "format": "prettier --write \"src/**/*.{js,jsx}\"",
+    "lint": "eslint \"src/**/*.{js,jsx}\" --quiet",
```
Comment (blacksrc) [inline package.json:6]: Why are you passing`--quiet` flag?

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "ecmaVersion": 2022,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
```
Comment (blacksrc) [inline .eslintrc.json:25]: Why?

File: package.json (hunk 1 above)
Comment (mim24sd) [reply package.json:6]: To report errors only.

File: .eslintrc.json (hunk 2 above)
Comment (mim24sd) [reply .eslintrc.json:25]: By this part of ESLint, we just have global variables of ES6 ,browser and node.js .

Comment (blacksrc) [reply .eslintrc.json:25]: Why do you need node?

File: package.json (hunk 1 above)
Comment (blacksrc) [reply package.json:6]: Why do you want to skip warnings?

Comment (mim24sd) [reply package.json:6]: This part recommended by "React Videos",
He said warnings are not big deals and errors are more important so better to focus on them.

but if warnings are necessary I will delete that part.

Comment (blacksrc) [reply package.json:6]: That's ok for now. But sometimes warnings can be very beneficial. Thanks for the explanation 👍🏽

File: .eslintrc.json (hunk 2 above)
Comment (mim24sd) [reply .eslintrc.json:25]: Ill talk to you about this

**Pull request merged**